### DISCUSSION TITLE
🐛 | Adding Files to Folder. New Files not playing with associated RFID card

### DISCUSSION THREAD

**Discussion by ChrisWitt** [MiczFlor/RPi-Jukebox-RFID#2020] [Q&A] [answered]
### What I did

Had folder with two audio files. Assigned RFID Card. 

Worked as expected. Card started playing files in folder.

Added two more audio files.

### What happened

The new audio files where not played when using the RFID Card associated with the folder.

### I expected this to happen

All four files in the folder should be played by the card associated to the folder.

### Further information that might help

I was unable to regenerate the folders playlist via web gui.

Via ssh I accessed RPi-Jukebox-RFID/playlists 
The playlist with the folder name contained only the two files previously in the folder.
Rebooted the phoniebox hopping for the playlist to be updated/regenerated.
Playlist still contained only the initial two files.

I removed the playlist with the same name as the folder.
Rebooted the phoniebox.

Fixed the issue. 

The folder playlist was regenerated with all four files.

### Base image and version

```
pi@raspberrypi:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

### Branch / Release
master

### RaspberryPi version

```
cat /sys/firmware/devicetree/base/model
Raspberry Pi 2 Model B Rev 1.1
```

**Comment by s-martin** [comment, marked as answer]
You need to update mpc, see

https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Troubleshooting-FAQ#i-added-new-folders-with-audiofiles-but-the-playout-does-not-work
